brainglobe_atlasapi/atlas_generation/atlas_scripts/perens_lsfm_mouse.py

```diff
@@ -17,15 +15,27 @@
     create_region_mesh,
 )
 from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.config import DEFAULT_WORKDIR
 from brainglobe_atlasapi.structure_tree_util import get_structures_tree
 
+### Settings
 PARALLEL = True  # disable parallel mesh extraction for easier debugging
 
+### Metadata
+__version__ = "2"
 
-### Additional functions #####################################################
+ATLAS_NAME = "perens_lsfm_mouse"
+SPECIES = "Mus musculus"
+ATLAS_LINK = "https://github.com/Gubra-ApS/LSFM-mouse-brain-atlas"
+CITATION = "Perens et al. 2021, https://doi.org/10.1007/s12021-020-09490-8"
+ORIENTATION = "ial"
+ROOT_ID = 997
+ATLAS_FILE_URL = (
+    "https://github.com/Gubra-ApS/LSFM-mouse-brain-atlas/archive/master.tar.gz"
+)
+RESOLUTION = 20  # some resolution, in microns
```

Comment:
I don't understand the comment - my best guess is that it's a relic from someone explaining the template script to themselves? If you agree, @viktorpm let's remove (if not, please clarify the comment)

Reply:
Your guess is correct 🙂. I'll remove it.